catalog/resource_catalog.go

```diff
@@ -8,6 +8,10 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
+func ucDirectoryPathSuppressDiff(k, old, new string, d *schema.ResourceData) bool {
+	return (new == (old + "/")) || (old == (new + "/"))
+}
+
```

Comment:
Thanks for the PR, can we add a comment mentioning why this was required? and add a log similar to the one below if it's true:

`log.Printf("[DEBUG] Ignoring configuration drift from %s to %s", old, new)`